BetterPasswordPlugin.inc.php

```diff
@@ -158,25 +158,15 @@ function checkPassword($hookName, $args) {
 			return false;
 		}
 		if ($this->getSetting(CONTEXT_SITE, 'betterPasswordCheckBlacklist')) {
-			$badPassword = false;
```

Comment:
The delete of this variable assignment seems unintentional, and will cause an undefined variable warning on 181/171.